R/formatter.r

```diff
@@ -1,3 +1,84 @@
+#' Number formatter: a generic formatter for numbers
+#'
+#' @return `number_format` returns a function with single parameter
```

Comment:
Can you please ident subsequent lines with two spaces? http://style.tidyverse.org/documentation.html#indents-and-line-breaks

Reply:
corrected